spec.html

```diff
@@ -48001,16 +48586,20 @@ <h1>
       </emu-annex>
 
       <emu-annex id="sec-parsepattern-annexb">
-        <h1>Static Semantics: ParsePattern ( _patternText_, _u_ )</h1>
+        <h1>Static Semantics: ParsePattern ( _patternText_, _u_, _v_ )</h1>
         <p>The semantics of <emu-xref href="#sec-parsepattern"></emu-xref> is extended as follows:</p>
-        <p>The abstract operation ParsePattern takes arguments _patternText_ (a sequence of Unicode code points) and _u_ (a Boolean). It performs the following steps when called:</p>
-        <emu-alg>
-          1. If _u_ is *true*, then
-            1. Let _parseResult_ be ParseText(_patternText_, |Pattern[+UnicodeMode, +N]|).
+        <p>The abstract operation ParsePattern takes arguments _patternText_ (a sequence of Unicode code points), _u_ (a Boolean), and _v_ (a Boolean). It performs the following steps when called:</p>
+        <emu-alg>
+          1. If _v_ is *true* and _u_ is *true*, then
+            1. Let _parseResult_ be a List containing one *SyntaxError* object.
+          1. Else if _v_ is *true*, then
+            1. Let _parseResult_ be ParseText(_patternText_, |Pattern[+UnicodeMode, +UnicodeSetsMode, +N]|).
+          1. Else if _u_ is *true*, then
+            1. Let _parseResult_ be ParseText(_patternText_, |Pattern[+UnicodeMode, ~UnicodeSetsMode, +N]|).
           1. Else,
```

Comment:
it seems like 80% of the steps in the two ParsePatterns are the same. Should perhaps annex b use step modification instead of replacing the entire AO wholesale?